renderdoc/driver/ihv/arm/official/lizard/thirdparty/hwcpipe/.clang-format

```diff
@@ -0,0 +1,107 @@
+---
```

Comment:
Please remove files like .clang-format and .gitignore from the third party library folders, as well as the library's own `CMakeLists.txt`.